backend/api/flask/endpoints/admin_config.py

```
@@ -6,20 +6,24 @@
 import logging
 
 logger = logging.getLogger(__name__)
 
 admin_config_bp = Blueprint("admin_config", __name__)
 
+_backend_require_session = require_identity_session(
+    get_redis_store=lambda: current_app.container.redis_kv_store,
+)
+
 
 def _get_current_user(req):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_get_current_user`

(ANN202)